EMIP/1-1000/EMIP00166.xml

```diff
@@ -38,9 +39,73 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                </msIdentifier>
                <msContents>
                   <summary>Homiliary in Honor of the Archangel Michael, ድርሳነ፡ ሚካኤል፡; Asmat Prayer of the Archangel Michael; Image of Michael, መልክአ፡ ሚካኤል፡</summary>
-                  <msItem xml:id="ms_i1">
                      <textLang mainLang="gez">Gə‘əz</textLang>
-                  </msItem>
+                  <msItem xml:id="ms_i1"><locus from = "1r" to = "103r"/><title type = "complete" ref = "LIT1295Dersan"/>
```

Comment:
see above. please note that if you are using Oxygen XML, you can use codetemplates to enter the msItem structure